src/matlab2tikz.m

```diff
@@ -407,9 +407,19 @@ function matlab2tikz(varargin)
 end
 % ==============================================================================
 function fid = fileOpenForWrite(m2t, filename)
-    encoding = switchMatOct({'native', m2t.cmdOpts.Results.encoding}, {});
```

Comment:
In the old way the third parameter would become `{}` for octave. This makes Octave 4 fail.